<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageresizer</class>
 <widget class="QDialog" name="imageresizer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imageresizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/icons/image_edit.ico</normaloff>:/icons/image_edit.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="urlHLayout">
     <item>
      <widget class="QLabel" name="sourceFileLabel">
       <property name="text">
        <string>Source File (s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Add Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="resolutionComboBox">
       <item>
        <property name="text">
         <string>VGA (640 x 480)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WVGA (800 x 480)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SVGA (800 x 600)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XGA (1024 X 768)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HD 720p (1280 x 720)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HD 1080p (1920 x 1080)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>640</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="byLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>by</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2160</number>
       </property>
       <property name="value">
        <number>480</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="debugTextBrowser">
     <property name="toolTip">
      <string>Browses to home directory to add images
</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="buttonBoxHLayout" stretch="3,1,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="statusProgressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
